package.json

```diff
@@ -38,7 +38,8 @@
   "defaults": {
     "roslyn": "4.8.0-1.23367.7",
     "omniSharp": "1.39.7",
-    "razor": "7.0.0-preview.23301.2"
+    "razor": "7.0.0-preview.23328.2",
+    "razorOmnisharp": "7.0.0-preview.23363.1"
```

Comment:
This looks like a newer version compared to above, but is actually built off an older version of the Razor language server (https://github.com/dotnet/razor/pull/8951).